cypress/pageObjects/assertions.ts

```diff
@@ -0,0 +1,22 @@
+import { eq } from "cypress/types/lodash"
+
+class Assertions {
+
+    checkHomePageIsOpen() {
+        cy.url().should('include', 'https://qaintazugg.restaurant365.com/#/grid/AllTransactionsGrid/00000000-0000-0000-0000-000000000000')
+
+    }
+
+    checkValidationMessageIsVisible() {
```

Comment:
you can send a flag variable here `checkValidationMessageIsVisible(isVisible: boolean)`
and the  method will be for both cases
`cy.contains("This Field Is Required").should(isVisible ? 'be.visible' : 'not.be.visible')`